src/main/kotlin/com/atlassian/performance/tools/sshubuntu/api/SshUbuntuContainer.kt

```diff
@@ -59,7 +66,10 @@ class SshUbuntuContainer(
      * https://github.com/testcontainers/testcontainers-java/issues/318
      * The class is a workaround for the problem.
      */
-    private class GenericContainerImpl(dockerImageName: String) : GenericContainer<GenericContainerImpl>(dockerImageName) {
+    private class GenericContainerImpl(dockerFileContent: String) : GenericContainer<GenericContainerImpl>(
+        ImageFromDockerfile(/* dockerImageName = */ "ssh-ubuntu", /* deleteOnExit = */ false)
```

Comment:
Should we add comments with argument names? I may be biased here, but I don't find it useful and even it's not obvious for me how should I read it, because it's not something I'm familiar with.

Reply:
Maybe a better representation would be something like this
```suggestion
        ImageFromDockerfile(
            "ssh-ubuntu", // dockerImageName
            false // deleteOnExit
        )
```

Reply:
I usually use named parameters in kotlin and run intellij intention for that.
For java code you can only use 'Add names in comments to call arguments' and I often find the code easier to read with such names included.
I can remove it if you don't find it useful

Reply:
I'm not very opinionated here, but I think it would be good to decide on a single approach/algorythom when it comes to adding names to Java class arguments or not doing that. I'm not sure if we can do that as part of this PR.

I think for the purpose of this PR you should decide yourself if it's worth adding or not.

Reply:
If we add it, I suppose it should look similar to kotlin named parameters.
It makes no harm to leave it as it is and later we can discuss the general rule